src/main/java/com/microsoft/azure/functions/dihook/FunctionInstanceFactory.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.dihook;

```

Comment:
remove for now, in future:
```suggestion
package com.microsoft.azure.functions.spi.inject;

```